Replace '+' with '|' in sed. With '+' it complains about wrong flag
--- ni/src/ncl/yMakefile.orig	2019-02-28 00:44:39
+++ ni/src/ncl/yMakefile	2025-02-19 14:06:24
@@ -655,12 +655,12 @@
 
 MakeNcl: MakeNcl.csh Makefile
 	@sed \
-	-e 's+SED_CC_LD+$(CC_LD)+' \
-	-e 's+SED_CC_OPT+$(CCOPTIONS)+' \
-	-e 's+SED_LDLIBS+$(LDLIBS)+' \
-	-e 's+SED_EXTRA_LDLIBS+$(EXTRA_LDLIBS)+' \
-	-e 's+SED_LIBPATH+$(LIBPATH)+' \
-	-e 's+SED_INCPATH+$(INCPATH)+' \
+	-e 's|SED_CC_LD|$(CC_LD)|' \
+	-e 's|SED_CC_OPT|$(CCOPTIONS)|' \
+	-e 's|SED_LDLIBS|$(LDLIBS)|' \
+	-e 's|SED_EXTRA_LDLIBS|$(EXTRA_LDLIBS)|' \
+	-e 's|SED_LIBPATH|$(LIBPATH)|' \
+	-e 's|SED_INCPATH|$(INCPATH)|' \
 	<MakeNcl.csh > MakeNcl
 
 #if defined(Darwin)
